src/RequestHandle.cpp

```diff
@@ -401,6 +446,19 @@ auto RequestHandle::onComplete() -> void {
     m_on_complete_handler(std::move(request));
 }
 
+auto RequestHandle::getRemainingDownloadBytesOptional() -> std::optional<size_t>
```

Comment:
Optional in the function name is redundant, its in the function signature

Reply:
Removed!